skypy/power_spectrum/growth.py

```diff
@@ -1,6 +1,21 @@
 """Linear growth module.
+
```

Comment:
There are two Documentation warnings: ` Unknown section Reference`. I think it should be `References`, plural. Is it possible to change it?